app/provider/app.go

```diff
@@ -477,18 +477,6 @@ func New(
 		authtypes.FeeCollectorName,
 	)
 	providerModule := ibcprovider.NewAppModule(&app.ProviderKeeper)
-
```

Comment:
@danwt I should have catch this while reviewing #84

Reply:
Hey, I'm not exactly sure what you mean. Can you elaborate a little?